test/dlc_tests/ecs/pytorch/inference/test_ecs_pytorch_inference.py

```diff
@@ -26,6 +26,28 @@ def test_ecs_pytorch_inference_cpu(pytorch_inference, ecs_container_instance, re
         ecs_utils.tear_down_ecs_inference_service(ecs_cluster_arn, service_name, task_family, revision)
 
 
+@pytest.mark.model("densenet")
+@pytest.mark.parametrize("ecs_instance_type", ["c5.4xlarge"], indirect=True)
```

Comment:
nit: it's fairly clear from the name, but can you also add `@pytest.mark.integration("elastic_inference")`